server/src/internalClusterTest/java/org/elasticsearch/search/routing/SearchReplicaSelectionIT.java

```diff
@@ -86,17 +85,13 @@ public void testNodeSelection() {
         assertNotNull(nodeStats);
         assertEquals(3, nodeStats.getAdaptiveSelectionStats().getComputedStats().size());
 
-        String bestNodeId = null;
-        double bestRank = Double.POSITIVE_INFINITY;
-        for (Map.Entry<String, ComputedNodeStats> entry : nodeStats.getAdaptiveSelectionStats().getComputedStats().entrySet()) {
-            double rank = entry.getValue().rank(1L);
```

Comment:
Before we fixed #71022, we passed in 1 for the number of outstanding requests so that this rank calculation matched the one used to select shards. Now that the bug is fixed, we need to remove the workaround.